javascript/javascript2/.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
You should add `.vscode` to your `.gitignore` file to avoid keeping these files in your repo.